robottelo/config/base.py

```diff
@@ -306,13 +306,15 @@ class DockerSettings(FeatureSettings):
     """Docker settings definitions."""
     def __init__(self, *args, **kwargs):
         super(DockerSettings, self).__init__(*args, **kwargs)
-        self.unix_socket = None
```

Comment:
i kept it, it's just down there on the line no. 313 ..you know, I didn't want to get NACK because of the alphabetical order :)))